src/main/java/ro/Homework/utility/StudentValidationException.java

```diff
@@ -0,0 +1,7 @@
+package ro.Homework.utility;
+
+public class StudentValidationException extends Exception {
```

Comment:
like we discussed during the course, checked exceptions should be avoided. Better to use unchecked exceptions
```suggestion
public class StudentValidationException extends RuntimeException {
```